examples/gettingStarted/scheduledMessages.ts

```diff
@@ -0,0 +1,103 @@
+/*
+  This sample demonstrates how .scheduleMessage() function can be used to schedule messages to appear
+  on a Service Bus entity at a specified later time.
+*/
+
+import {
+  Namespace,
+  SendableMessageInfo,
+  OnMessage,
+  MessagingError,
+  OnError,
+  ServiceBusMessage
+} from "../../lib";
+import { delay } from "rhea-promise";
```

Comment:
This would need the user to have `rhea-promise` as a dependency, which wouldnt be ideal when we want to enable them to just copy/paste the code.

We should figure out an alternative

Reply:
Will address this in a different issue